### DISCUSSION TITLE
Impressed after 5 Minutes

### DISCUSSION THREAD

**Discussion by DanielBednorz** [community/community#14241] [Copilot Conversations]
I started using Copilot today and have to say, that I'm impressed. Took me 5 minutes to that state :D
Saved me ~10minutes for writing tests. Those 2 suggestions where nearly perfect. What was missing to get a perfect match was, that a constructor was used that isn't present. In that case usage of the setter would be better. But this is whining on a high level.

I just want to say: Thank you!!

**Comment by Urethramancer** [comment]
I get great results now. Feels like it's adopted to my project layout and naming conventions, when initially it suggested naming based on bad ideas from the 'net. It also remembers closing handles I keep forgetting.
